tasks/create-advisory/create-advisory.yaml

```diff
@@ -55,10 +55,10 @@ spec:
         RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/create-advisory-results.json"
 
         # Obtain application from snapshot
-        application=$(jq -rc .application "$(workspaces.data.path)/$(params.snapshotPath)")
+        application=$(jq -r .application "$(workspaces.data.path)/$(params.snapshotPath)")
```

Comment:
`-r` doesn't make sense here - it's an object, not just a string. So if anything, I would change it to `-c`.

Reply:
No it isn't, it is a string https://github.com/konflux-ci/application-api/blob/main/api/v1alpha1/snapshot_types.go#L28

Reply:
Ah, sorry.